_observing-your-data/ad/dashboards-anomaly-detection.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+title: Anomaly detection dashboards and visualizations
```

Comment:
This still doesn't sound right to me... maybe 'Anomaly detection in dashboards and visualizations'?

Reply:
Do we need 'dashboards' at all? Is it assumed you have visualizations in your dashboards? Also, we don't call out adding anomaly detection to dashboards here, only visualizations.

Reply:
Dashboards and visualizations are separate topics. The use case is creating dashboards that show you how your system is running. A dashboard includes visualizations and data about the anomaly details.